docker/scripts/zkevm-prover.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eEu -o pipefail
+
+if [[ -v CONFIG_SECRET_NAME ]]; then
+  echo "Loading config file from AWS secrets manager"
+   aws secretsmanager  get-secret-value --secret-id ${CONFIG_SECRET_NAME} --query SecretString --output text | tee /app/config.json
```

Comment:
Should we be `tee`ing this if it's a secret?

Reply:
We should definitely not be echoing that into stdout. Let me fix that..